bodhi/models/models.py

```diff
@@ -1649,6 +1649,27 @@ def check_requirements(self, session, settings):
 
         return True, "All checks pass."
 
+    @property
+    def has_negative_karma(self):
+        """Check for negative karma, Returns True if the update has negative karma"""
+        for comment in self.comments:
+            if not comment.anonymous:
+                if comment.karma < 0:
+                    return True
```

Comment:
Your previous implementation of this method seemed to take into account the possibility of a user giving an accidental -1 and then a later +1. It seems that this code no longer handles that possibility, and will count accidental -1's and mark it as negative karma.